bin/cron.py

```diff
@@ -89,6 +89,7 @@ def update_security_advisories():
 
 
 @scheduled_job('interval', minutes=5)
+@ping_dms
```

Comment:
Should also remove the one decorating `update_product_details`. No need to ping twice every half hour.